http-server/configuration-prerelease.yml

```diff
@@ -1,2 +1,34 @@
 githubRepo: test
-githubApiToken: ${GITHUB_ISSUES_API_TOKEN}
+githubApiToken: ${GITHUB_API_TOKEN:-test}
```

Comment:
The ':-test' syntax defaults a missing env variable value to 'test' here. We do not have that defaulting active in the prod configuration.

Otherwise the updates here are mostly a copy/paste of me googling what it took to configure database in drop wizard. The values for the most part looked reasonable.

Of note we still bake in the fact/notion that database must be running on the same machine, I believe we have this assumption for lobby and it's a bit of a long standing TODO to allow for lobby, http-server and DB to be running on different machines. For now we intend a single-machine deployment, so it's okay..